src/pages/app.jsx

```
@@ -1,40 +1,45 @@
-import { DailyWeatherData } from '../components/dailyWeather';
-import Changelog from './changelog.jsx';
+
+// Lazy load all components
+const Home = lazy(() => import('./home.jsx'));
+const About = lazy(() => import('./about.jsx'));
+const Weather = lazy(() => import('./weather.jsx'));
+const GetSingleWeather = lazy(() => import('../components/weatherSingle.jsx').then(module => ({ default: module.GetSingleWeather })));
+const ThreeHourForecastData = lazy(() => import('../components/3HourForecastData.jsx').then(module => ({ default: module.ThreeHourForecastData })));
+const ShowMap = lazy(() => import('../components/showMap'));
+const SingleThreeHourForecastData = lazy(() => import('../components/Single3HourForecastData').then(module => ({ default: module.SingleThreeHourForecastData })));
+const Error = lazy(() => import('./error'));
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)